paper-checkbox.html

```diff
@@ -136,6 +138,17 @@
       }
     },
 
+    // button-behavior hook
+    _buttonStateChanged: function() {
+      this.checked = this.active;
+    },
+
+    _checkedChanged: function(checked) {
+      this.setAttribute('aria-checked', this.checked ? 'true' : 'false');
+      this.active = this.checked;
```

Comment:
Are `active` and `checked` synonymous? We've gone back and forth on this a bit, so I'm just curious if we settled on something.